Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.Data.cs

```diff
@@ -19,20 +20,18 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls.WrapPanel
     /// </summary>
     public partial class WrapPanel
     {
-        internal class UvMeasure
+        [System.Diagnostics.DebuggerDisplay("U = {U} V = {V}")]
+        private struct UvMeasure
         {
-            internal double U { get; set; }
-
-            internal double V { get; set; }
+            private double u, v;
+            internal double U { get { return u; } set { u = Math.Floor(value); } }
```

Comment:
There should be a space between private double u, v;  and the next line.
Internal double U .... need to be on multiple lines.
The toolkit already has StyleCop file that contains everything regarding the recommended practices which should be highlighted in the file by default.